### ISSUE TITLE
Feature request: binding for "copy screenshot of a page to a clipboard"

### ISSUE THREAD

**Issue by eyehol3** [brookhong/Surfingkeys#1267]
it's nice to have a binding to make a screenshot but you have to click on a picture with a mouse and look for "copy image" button. it's somewhat painful and almost defies purpose: it's faster to manually do a screenshot for now.

**Comment by ngochangjelly** [comment]
@eyehol3  yes, I would love it so much to have this feature. I try to implement it by myself, by copying base64 to clipboard and find a way to convert that base64 to picture when the user paste out, but it hasn't worked yet 😢
